src/m1_facilitator/SeedDataInitializer.ts

```diff
@@ -0,0 +1,197 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Web3 from 'web3';
+import * as Mosaic from 'Mosaic';
+import BigNumber from 'bignumber.js';
+
+import Gateway, { GatewayType } from './models/Gateway';
+import Anchor from './models/Anchor';
+import Repositories from './repositories/Repositories';
+import ContractEntity, { EntityType } from '../common/models/ContractEntity';
+import Utils from '../common/Utils';
```

Comment:
Let's sort imports.

Reply:
I can do this but is it part of airbnb style guide? Are we doing this everywhere?